### PR TITLE
MVP-101: Added github issues endpoint

### PR DESCRIPTION
**Pull request by BradyMitch**
## 🎯 Summary

  
[MVP-101](https://citz-imb.atlassian.net/browse/MVP-101)

Endpoint gives information on a GitHub repo's issues through use of the github api.

Test with Postman or ThunderClient (VSCode Extension) or similar by making a `GET` request to `http://localhost:6600/github/issues/citz-imb-common-code?state=open` while running the backend container.

Also fixed getParams and getQuery so they now return the correct error response.

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](/CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - I have consulted with the team if introducing a new dependency.
> - My changes generate no new warnings.

### CODE REVIEW

File: backend/src/modules/github/controller.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { Request, Response } from 'express';
+import { HttpError, errorWrapper, getParams, httpStatusCode } from '../../utils';
+import { getOpenIssuesParamsSchema } from './schemas';
+
+/**
+ * Use github api to lookup open issues for a repo.
+ * @method GET
+ * @route /github/issues/open/:repo
+ */
+export const getOpenIssues = errorWrapper(async (req: Request, res: Response) => {
```
Comment (BradyMitch) [inline backend/src/modules/github/controller.ts:10]: Should we change this to `getIssues` and add a query parameter for `state` which we can just pass `open`?

Comment (BradyMitch) [reply backend/src/modules/github/controller.ts:10]: @TaylorFries @SDToews

Comment (TaylorFries) [reply backend/src/modules/github/controller.ts:10]: that or if we wanted to get closed issues we could add getClosedIssues in the future, OR in the future we can add a getClosedIssues and getAll which would call both and then merge them.

**Review by TaylorFries — APPROVED** [review]
endpoint works as expected. no issues detected. 
and it looks great.

**Review by TaylorFries — APPROVED** [review]
query works as expected, no issues detected

**Pull request merged**